package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.37.0",
+  "version": "7.37.1",
```

Comment:
We've generally tried to separate version bumps into their own pull-requests, instead of bundling them with other changes. For example:

https://github.com/workos/workos-node/commit/207686f530c03251d79729f41ade53f6725c3dd7